docs/concepts/policy/pod-security-policy.md

```diff
@@ -125,141 +399,153 @@ to the volume sources that are defined when creating a volume:
 The recommended minimum set of allowed volumes for new PSPs are
 configMap, downwardAPI, emptyDir, persistentVolumeClaim, secret, and projected.
```

Comment:
I'd highlight the recommended volume types.

Reply:
Done.